e2e-tests/playwright/utils/common.ts

```diff
@@ -305,7 +305,7 @@ export class Common {
         const authorization = popup.locator("button.js-oauth-authorize-btn");
         if (await authorization.isVisible()) {
           authorization.click();
-          return "Login successful with app authorization";
+          return "Login successful";
```

Comment:
@albarbaro I updated this because this check fails under [this case](https://github.com/redhat-developer/rhdh/blob/753eaeb79c8660cf13dc244062f88c856b5928db/e2e-tests/playwright/utils/common.ts#L308), PTAL :)

Reply:
Looks good to me! Thanks!